setup.py

```diff
@@ -17,11 +17,11 @@
         long_description = f.read()
 
     install_requirements = [
-        'vivarium>=0.8.19',
+        'vivarium==0.8.20',
         # FIXME: Newer versions of numpy have conflicting dependencies
         # with pytables.
         'numpy<=1.15.4',
-        'pandas',
+        'pandas<0.25',
```

Comment:
I'm not sure we want to pin pandas here actually. The pin is because of a central comp dependency that only affects data so it doesn't seem like it needs to be pinned here